frontend/src/components/auth/Signup.tsx

```diff
@@ -56,21 +63,28 @@ const Signup = (): React.ReactElement => {
     setSignupErrors(newSignupErrors);
   };
 
+  // eslint-disable-next-line consistent-return
   async function onSubmit(event: React.FormEvent<HTMLFormElement>) {
     event.preventDefault();
     if (!isLastStep) return next();
-    const user: AuthenticatedUser = await register({ ...data });
-    if (user) {
-      alert("Successful sign up!");
-    } else {
+    try {
+      const user: AuthenticatedUser = await register({ ...data });
+      if (user) {
```

Comment:
We should check to see what role the user is and then redirect them to the appropriate page.

The logic is here in `Login.tsx`
```
if (authenticatedUser) {
    if (authenticatedUser.role === "VOLUNTEER") {
      return <Redirect to={VOLUNTEER_DASHBOARD_PAGE} />;
    }
    if(authenticatedUser.role === "ADMIN"){
      return <Redirect to={PLATFORM_SIGNUP_REQUESTS} />;
    }
    return <Redirect to={HOME_PAGE} />;
  }
  ```

Reply:
Also experiment with `<Redirect>` as opposed to history.push(). I don't mind the latter but if we can make redirect work I think it fits the pattern better.